src/components/v5/common/CountDownTimer/CountDownTimer.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import Icon from '~shared/Icon';
+
+import { MiniSpinnerLoader } from '~shared/Preloaders';
+import TimerValue from '~shared/TimerValue';
+
+import { CountDownTimerProps } from './types';
+
+const displayName = 'common.CountDownTimer';
+
+const CountDownTimer: FC<CountDownTimerProps> = ({
+  countdown,
+  loadingCountdown,
+}) => {
+  if (loadingCountdown) {
+    return <MiniSpinnerLoader />;
+  }
+
+  return (
+    <div data-test="countDownTimer" className="flex items-center gap-1">
+      <Icon
+        name="clock"
+        className="text-negative-400 [&_svg]:h-[1em] [&_svg]:w-[1em] [&_svg]:text-[.875rem]"
+      />
+      <span className="text-4 text-gray-900">
```

Comment:
is` text-gray-900 ` needed?

Reply:
Yes, it sets the color of the text. I want to avoid a situation where someone is setting a different color on a wrapper and the text color would be inherited by this component.